scripts/quests/ahtUrhgan/Rock_Bottom.lua

```diff
@@ -2,47 +2,62 @@
 -- Rock Bottom
 -- !pos 838.243 -14.475 231.871 61
 -----------------------------------
-require("scripts/globals/items")
-require("scripts/globals/quests")
-require("scripts/globals/npc_util")
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/npc_util')
 require('scripts/globals/interaction/quest')
 -----------------------------------
 
 local quest = Quest:new(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.ROCK_BOTTOM)
 
-quest.sections = {
-
+quest.sections =
+{
+    -- Section: Quest available
     {
         check = function(player, status, vars)
             return status == QUEST_AVAILABLE
         end,
 
-        [xi.zone.MOUNT_ZHAYOLM] = {
-            ['11'] = {
+        [xi.zone.MOUNT_ZHAYOLM] =
+        {
+            ['11'] =
```

Comment:
Just checked npc_list and... No.
Unless it's supposed to be qm11. I'll test it whenever I have time. It may be just a missing qm or it may be one of those "blank" npcs that need a renaming.

Reply:
It is indeed a "blank" npc. Went with renaming it to blank11 in both sqk and scripts